packages/shared/src/components/auth/AuthModalFooter.tsx

```diff
@@ -15,7 +16,10 @@ function AuthModalFooter({
   onIsLogin,
 }: AuthModalFooterProps): ReactElement {
   return (
-    <Modal.Footer className={className} justify={Justify.Center}>
+    <Modal.Footer
+      className={classNames(className, 'gap-[unset]')}
```

Comment:
I think it might be more efficient to add it to the tailwind config, these bracket once as I remember add more bloat stuff.

Reply:
Good point. At a later stage, we might want to take a look at the ModalFooter on why it has a `gap-3` built-in.